### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by 157243321** [langgenius/dify#8762] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Hello! 
Could you please tell me how to change `http://df.*********.com/v1` to `https://df.********.com/v1`? This is because if I use the API call, it doesn't work due to the missing "S". Where should I make the modification in the documentation?      

  
你好！
请问我如何把  http://df.********.com/v1
               换成 https://df.********.com/v1
因为如果我用API调用，无法使用。
因为少了"S",我应该在文档那个地方进行修改呢？
谢谢！
![微信截图_20240925175934](https://github.com/user-attachments/assets/b300fe44-3880-4ab7-886a-59d7e0b3d080)

**Comment by cgthunderstrike** [comment]
help
